templates/site/js/login-styles-scripts.js

```diff
@@ -0,0 +1,41 @@
+/*Initialize Popper.js and Tooltips.js*/
+$(document).ready(function() {
+  $('[data-toggle="tooltip"]').tooltip();
+  $('[data-toggle="popover"]').popover();
+
+})
+
+/*Toggle eye icon and password text in password input field*/
+function togglePassword() {
+    var passwordInput = document.getElementById("userPassword");
+
+    if (passwordInput.type === "password") {
+        passwordInput.type = "text";
+    } else {
+        passwordInput.type = "password";
+    }
+}
+
+function toggleEyeIcon(element) {
+    var eyeIcon = element.firstElementChild;
+
+    eyeIcon.classList.toggle("icon-eye");
+    eyeIcon.classList.toggle("icon-eye-slash");
+}
+
+
+window.onload = () => {
```

Comment:
Is there a reason why you're doing this here instead of with the `$(document).ready` code above? (I know that they trigger at slightly different times but does that matter in this case?)

Reply:
It was just developed before I got tooltips working and needed to import jquery.

Reply:
Ok makes sense. They should be merged now

Reply:
Moved this functionality into jquery initialize function.